src/Entities/RegistryContext.h

```diff
@@ -0,0 +1,45 @@
+#include <Graphics/DebugLines.h>
+
+namespace openblack::Entities {
+class RenderContext
+{
+public:
```

Comment:
A class with everything public is a struct. You could use a struct here

Reply:
I'm going to add a constructor and move _dirty and _hasBoundingBoxes as well. Don't see anything wrong with them being structs.